grails-app/views/order/show.gsp

```diff
@@ -282,6 +282,17 @@
                     },
                     ajaxOptions: {
                       error: function(xhr, status, index, anchor) {
+                        var errorMessage = "Error loading tab: " + xhr.status + " " + xhr.statusText;
```

Comment:
I assume it's copied from elsewhere, but
`var` -> `let`

Reply:
fixed